cogs/help.py

```diff
@@ -34,6 +34,7 @@
 CUSTOM_ID_PREFIX: str = "help:"
 NAME_TOPIC_REGEX: str = r"^(?P<topic>.*?) \((?P<author>[^)]*[^(]*)\)$"
 WAIT_FOR_TIMEOUT: int = 1800  # 30 minutes
+NO_HELP_MESSAGE: str = "You are banned from creating help threads."
```

Comment:
I feel this can be elaborated a bit

Reply:
Why was this marked as resolved without any action?
Please give a response before marking reviews as resolved.

Reply:
wait somehow i misclicked :(

Reply:
I agree, possibly some form of action to take if they feel its unfair. Like dm'ing modmail